src/lib.rs

```diff
@@ -216,7 +217,7 @@ pub use crate::error::Error;
 // The function MUST NOT ever write uninitialized bytes into `dest`,
 // regardless of what value it returns.
 cfg_if! {
-    if #[cfg(any(target_os = "haiku", target_os = "redox", target_os = "nto", target_os = "aix"))] {
+    if #[cfg(any(target_os = "haiku", target_os = "hurd", target_os = "redox", target_os = "nto", target_os = "aix"))] {
```

Comment:
We either should use `all(target_os = "hurd", target_env = "gnu")` or use `*-hurd-*` in the docs table.

Reply:
ok, used `*-hurd-*`